src/index.js

```diff
@@ -15,7 +16,7 @@ image.classList.add('refresh-icon');
 image.setAttribute('src', fresh);
 textContainer.appendChild(image);
 
-const deleteBtn = document.querySelector('.delete-btn');
+// const deleteBtn = document.querySelector('.delete-btn');
```

Comment:
- [ ] To keep your code clean, please remove unused commented codes from it. Thank you :)